Src/IronPython/Runtime/MemoryView.cs

```diff
@@ -930,7 +931,7 @@ public sealed class MemoryView : ICodeFormattable, IWeakReferenceable, IBufferPr
                 throw PythonOps.BufferError("memoryview: underlying buffer is not writable");
 
             if (flags.HasFlag(BufferFlags.CContiguous) && !_isCContig)
-                throw PythonOps.BufferError("memoryview: underlying buffer is not C-contiguous");
+                throw PythonOps.BufferError("memoryview: underlying buffer is not C contiguous");
```

Comment:
I thought Python used "C-contiguous" with a dash?
```py
struct.unpack("H", memoryview(b"AA")[::-1])
```
> BufferError: memoryview: underlying buffer is not C-contiguous

Reply:
`memoryview` does, but `ctypes` doesn't.
```python
c_short.from_buffer(memoryview(bytearray(b"AA"))[::-1])
```
> TypeError: underlying buffer is not C contiguous

Unfortunately, tests for `ctypes` check for this form. Will see if I can get it both ways.

Reply:
Took a peek at the CPython ctypes implementation and it seems like `from_buffer` is the odd one:
```
ctypes.c_int.from_buffer(memoryview(bytearray(b"aaaa"))[::-1])
```
> TypeError: underlying buffer is not C contiguous

but `from_buffer_copy` gives the `memoryview` error:
```
ctypes.c_int.from_buffer_copy(memoryview(bytearray(b"aaaa"))[::-1])
```
> BufferError: memoryview: underlying buffer is not C-contiguous